7 ftp using the universal mounting system (#31)
opulo-inc/docs: content/en/openpnp/calibration/connect-to-machine/index.md

Awesome! Now we've got OpenPnP installed on your computer, and we've got the default configuration loaded up, our next step is getting connected to the machine using OpenPnP and getting familiar with the UI.

1. Connect your machine to your computer using the included USB-C cable, and open OpenPnP on your computer. You should now see the OpenPnP UI:

{{< container-image path="images/openpnp-ui.png" alt="" >}}

## Com Port and Baud Rate

> Before we can connect to the machine, we need to tell OpenPnP which USB ports to use for communication with your LumenPnP.

Reviewer: ```suggestion
Before connecting to the machine, you need to set which USB ports to use for communication in OpenPnP.
```

18. Optional: in the `Format` drop-down, choose the `1280x720 10fps` setting

19. Click the "Apply" button in the bottom right
  {{< container-image path="images/Top Camera Apply.png" alt="Saving changes to the Top Camera Config" >}}

20. To check that both cameras are working now, select your preferred view option in the top left `Cameras` section. The Camera feeds should update automatically with a very slight delay.
  {{< container-image path="images/Both-cameras-working.png" alt="Both cameras showing live outputs." >}}

## Connecting to the LumenPnP

21. Click the green power button in the Machine Controls section of the UI to connect to your machine.
  {{< container-image path="images/Screen Shot 2022-05-17 at 4.14.47 PM.png" alt="" >}}

22. The power button should turn red, and OpenPnP has connected to your machine. You should also see both webcam feeds as shown below. (If you don't see any, or just one, under the `Cameras` section, select the `Show All Horizontal` option from the dropdown.)
  {{< container-image path="images/Screen Shot 2022-05-17 at 4.16.20 PM.png" alt="" >}}

> 23. While homing won't work properly until we finish calibrating the fiducial locations, you should now be able to manually "jog" your LumenPnP. Select 1mm or 10mm on the Distance slider, and then press the arrow buttons to move each axis of your machine. Again, since the machine isn't properly homed it may not move all the way to its boundaries. This is simply a test to make sure we can send it Gcode successfully.

Reviewer: ```suggestion
23. While homing won't work properly until you finish calibrating the fiducial locations, you should now be able to manually "jog" your LumenPnP. Select 1mm or 10mm on the Distance slider, and then press the arrow buttons to move each axis of your machine. Again, since the machine isn't properly homed it may not move all the way to its boundaries. This is simply a test to make sure that OpenPnP can send it Gcode successfully.
```
